server.elm

```diff
@@ -379,6 +379,20 @@ if iscloseable then <span dummy=""></span> else closeEditBox,
   transition: .4s;
 }
 
+/* Remove input default border and shadow */
+input[type=text] {
```

Comment:
Can you please make this selector more specific?
E.g. prepend it with div#modify-menu